book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
```

Comment:
- [ ] Let's add `:rental` to attr_accessor as users can write and read the rental. See suggestion below:

```suggestion
  attr_accessor :title, :author, :rental
```

Reply:
Thanks for the suggestion I'll implement it!